src/test/kotlin/com/terraformation/backend/accelerator/db/ApplicationStoreTest.kt

```diff
@@ -315,6 +323,418 @@ class ApplicationStoreTest : DatabaseTest(), RunsAsUser {
+                submissionStatus = SubmissionStatus.InReview,
+            )
+      }
+
+      @Test
+      fun `filters by IDs`() {
+        val deliverableSubmissionModel1 =
+            DeliverableSubmissionModel(
+                category = DeliverableCategory.Compliance,
+                deliverableId = prescreenDeliverableId,
+                descriptionHtml = "Pre-screen deliverable description",
+                documents = emptyList(),
+                dueDate = null,
+                feedback = "feedback 1",
+                internalComment = "comment 1",
+                moduleId = prescreenModuleId,
+                moduleName = "Pre-screen",
+                moduleTitle = null,
+                name = "Pre-screen deliverable",
+                organizationId = organizationId,
+                organizationName = "Organization 1",
+                participantId = null,
+                participantName = null,
+                projectId = org1ProjectId1,
+                projectName = "Project A",
+                status = SubmissionStatus.InReview,
+                submissionId = submissionId1,
+                templateUrl = null,
+                type = DeliverableType.Questions,
+            )
+
+        val deliverableSubmissionModel2 =
+            deliverableSubmissionModel1.copy(
+                category = DeliverableCategory.CarbonEligibility,
+                deliverableId = applicationDeliverableId,
+                descriptionHtml = "Application deliverable description",
+                feedback = "feedback 2",
+                internalComment = "comment 2",
+                moduleId = applicationModuleId,
+                moduleName = "Application",
+                name = "Application deliverable",
+                submissionId = submissionId2,
+            )
+
+        val deliverableSubmissionModel3 =
+            deliverableSubmissionModel1.copy(
+                feedback = "feedback 3",
+                internalComment = "comment 3",
+                projectId = org1ProjectId2,
+                projectName = "Project B",
+                submissionId = submissionId3,
+            )
+
+        val deliverableSubmissionModel4 =
+            deliverableSubmissionModel2.copy(
+                feedback = "feedback 4",
+                internalComment = "comment 4",
+                projectId = org1ProjectId2,
+                projectName = "Project B",
+                submissionId = submissionId4,
+            )
+
+        val deliverableSubmissionModel5 =
+            deliverableSubmissionModel1.copy(
+                feedback = "feedback 5",
+                internalComment = "comment 5",
+                organizationId = organizationId2,
+                organizationName = "Organization 2",
+                projectId = org2ProjectId1,
+                projectName = "Project C",
+                submissionId = submissionId5,
+            )
+
+        val deliverableSubmissionModel6 =
```

Comment:
Not a huge deal, but this test might be a little easier to read if these names included which org/project was involved. While reviewing I had to keep scrolling back to see which one was which.